src/windows/View.tsx

```diff
@@ -0,0 +1,371 @@
+/**
+* View.tsx
+*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT license.
+*
+* Windows-specific implementation of View.
+*/
+
+import React = require('react');
+import RNW = require('react-native-windows');
+import Types = require('../common/Types');
+import PropTypes = require('prop-types');
+
+import {View as ViewCommon} from '../native-common/View';
+import EventHelpers from '../native-common/utils/EventHelpers';
+import { applyFocusableComponentMixin, FocusManagerFocusableComponent, FocusManager } from '../native-desktop/utils/FocusManager';
+
+const KEY_CODE_ENTER = 13;
+const KEY_CODE_SPACE = 32;
```

Comment:
Should we define these somewhere common rather than repeating them across multiple files?

Reply:
Maybe, but not sure where.

---
In reply to: [159359132](https://github.com/Microsoft/reactxp/pull/404#discussion_r159359132) [](ancestors = 159359132)